### ISSUE TITLE
Add a heartbeat resizing effect on the sumo timer

### ISSUE THREAD

**Issue by SaltySnail** [SaltySnail/Sumo-BeamMP#4]
The timer needs to behave like the timer at the top of [this video](https://youtu.be/vdCwHfGFx9U?si=Rj226Cmy-ax90Gjh&t=62).
In `Client/ui/modules/apps/SumoTimer/app.js` change:
               `js
                let x = canvas.width / 2;
                let y = canvas.height / 2;
`
To `<width/height> / 3`.
When `Engine.Audio.playOnce('AudioGui', "/art/sound/timerTick", {volume = 60})` gets called in `Client/lua/ge/extensions/Sumo.lua`, the timer UI needs to be bigger (`<width/height> / 2`). 

The transitions between being small and big need to be smooth. This can be done using the `time` variable in `Client/ui/modules/apps/SumoTimer/app.js`, which is accurate on 10ms. 

To sum up: UI get big when sound plays, UI get small when halfway to next sound (sound plays last 5 seconds of a round)
